jwst/stpipe/step.py

```diff
@@ -1296,3 +1302,31 @@ def _get_suffix(suffix, step=None, default_suffix=None):
     if suffix is None and step is not None:
         suffix = step.name.lower()
     return suffix
+
+
+def get_disable_crds_steppars(default=None):
```

Comment:
```suggestion
def get_disable_crds_steppars(flag=None):
```

The name `default` for this arg was a bit misleading to me

Reply:
Of course, this would need to be refactored in the function as well